lambdas/check-existing-identity/src/test/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandlerTest.java

```diff
@@ -201,9 +202,10 @@ void shouldReturnJourneyReuseResponseIfScoresSatisfyM1AGpg45Profile() throws Exc
     @Test
     void shouldReturnJourneyResetIdentityIfResetIdentityFeatureFlagIsEnabled() throws Exception {
         when(ipvSessionService.getIpvSession(TEST_SESSION_ID)).thenReturn(ipvSessionItem);
-        when(userIdentityService.getUserIssuedCredentials(TEST_USER_ID)).thenReturn(CREDENTIALS);
+        when(userIdentityService.getUserIssuedCredentials((List<VcStoreItem>) any()))
+                .thenReturn(CREDENTIALS);
         when(criResponseService.getFaceToFaceRequest(TEST_USER_ID)).thenReturn(null);
-        when(userIdentityService.areVcsCorrelated(TEST_USER_ID)).thenReturn(true);
+        when(userIdentityService.areVCsCorrelated((List<VcStoreItem>) any())).thenReturn(true);
```

Comment:
Can we get rid of these type casts now?

Reply:
what's wrong in this ? 
or instead can use anyList()

Reply:
Now there's only one signature, I think the cast is unnecessary (gets a squiggle in IntelliJ). `anyList()` would be fine too though.

Reply:
done